doc/devel/contributing.rst

```diff
@@ -326,6 +326,14 @@ Introducing
    All these helpers take a first parameter *since*, which should be set to
    the next point release, e.g. "3.x".
 
+   When providing alternatives in ``_api.warn_deprecated()`` or
+   ``@_api.deprecated``, you can use backticks to generate links to these
```

Comment:
Maybe rephrase this whole paragraph as  "You can use standard rst cross references in the alternative.  The string will be processed to remove back ticks and only include the link caption in the runtime warning"  or something similar?  I think we can expect people who are deprecating functionality to be at least familiar with rst.  I think this is a case where trying to be verbose and clear makes it more confusing ;)

Reply:
Sure, sounds much better. (it's always difficult for me to be concise and clear at the same time when writing in English)